transformer/events_creator_test.go

```diff
@@ -157,7 +165,48 @@ func (s *EventsCreatorTestSuite) TestTransformer_ProcessAggregatedResources() {
 				s.mockedEvaluator.On(methodMock.methodName, methodMock.args...).Return(methodMock.returnArgs...)
 			}
 
-			transformer := NewTransformer(ctx, &s.mockedEvaluator, testIndex)
+			//Need to add services
+			kc := k8sfake.NewSimpleClientset()
+
+			namespace := &v1.Namespace{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "Namespace",
+					APIVersion: "apps/v1beta1",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: tt.args.namespace,
+				},
+			}
+
+			node := &v1.Node{
+				TypeMeta: metav1.TypeMeta{
+					Kind: "Node",
+					APIVersion: "apps/v1beta1",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "testing_node",
+					UID: "testing_node_uid",
+				},
+			}
+
+			_, err := kc.CoreV1().Namespaces().Create(ctx, namespace, metav1.CreateOptions{})
+			s.NoError(err)
+
+			_, err = kc.CoreV1().Nodes().Create(ctx, node, metav1.CreateOptions{})
+			s.NoError(err)
+
+			cdp := CommonDataProvider{
+				kubeClient: kc,
+				fsys: fstest.MapFS{
+					"hostname": {
+						Data: []byte("testing_node"),
+					},
+				},
+			}
+
+			transformer, err := NewTransformer(ctx, &s.mockedEvaluator, cdp, testIndex)
+			s.NoError(err)
+		
```

Comment:
what about checking the relevant field in the for loop of this test?
or in a separate test?